.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v4.2.0
+      hooks:
+        -   id: trailing-whitespace
+        -   id: end-of-file-fixer
+        -   id: mixed-line-ending
+            name: Check line ending character (LF)
+            args: ["--fix=lf"]
+            types_or: [python, c, c++, shell]
```

Comment:
What happens if you remove this? 

Right now it's ignoring markdown.

Reply:
I removed `types_or` as you suggested, it shouldn't ignore markdown anymore.